src/test/java/org/takes/tk/TkSlf4jTest.java

```diff
@@ -55,4 +61,36 @@ public void logsException() throws IOException {
         new TkSlf4j(new TkFailure(new IOException(""))).act(new RqFake());
     }
 
+    /**
+     * {@link TkSlf4j} can output an empty body for {@link TkEmpty}.
+     * @throws IOException if some I/O problem occurred.
+     */
+    @Test
+    public void testEmptyResponseBody() throws IOException {
+        new FtRemote(
+            new TkSlf4j(new TkEmpty())
+        ).exec(
+            new FtRemoteScriptPost()
+        );
+    }
+
+    /**
+     * {@link FtRemote.Script} that does a POST request with a "test" body.
+     * @author Rui Castro (rui.castro@gmail.com)
+     * @version $Id$
+     * @since 0.43
+     */
+    private class FtRemoteScriptPost implements FtRemote.Script {
+        @Override
+        public void exec(final URI home) throws IOException {
+            new JdkRequest(home)
+                .method("POST")
+                .body().set("test").back()
+                .fetch()
+                .as(RestResponse.class)
+                .assertBody(new IsEqual<>(""))
```

Comment:
@rui-castro What about `Matchers.isEmptyOrNullString()`  instead of `new IsEqual<>("")`?